CHANGELOG.md

```diff
@@ -14,6 +14,10 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## [2.42.1] - 2022-03-15
+### Added
```

Comment:
`Added` means new feature. I think it implies minor version bump. 2.43.0

Reply:
Thanks! I'll fix this, could you also approve this one so that this only depends on starbase approval?